.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default owners for repo
+* @caraml-dev/platform
```

Comment:
This means everything in the repo is owned by platform team. When we will add the sub-chart (say `charts/caraml-store`), we will add a new rule in this same file saying `charts/merlin/** @caraml-dev/caraml-store`. Am I right?

Reply:
yes, you are right